rxbinding/src/main/java/com/jakewharton/rxbinding/view/MenuItemClickEvent.java

```diff
@@ -0,0 +1,38 @@
+package com.jakewharton.rxbinding.view;
+
+import android.content.Context;
+import android.support.annotation.CheckResult;
+import android.support.annotation.NonNull;
+import android.view.MenuItem;
+
+/**
+ * A click event on a menu item.
+ * <p>
+ * <strong>Warning:</strong> Instances keep a strong reference to the menu item. Operators that
+ * cache instances have the potential to leak the associated {@link Context}.
+ */
+public final class MenuItemClickEvent extends MenuItemEvent<MenuItem> {
```

Comment:
I mentioned this on another PR, but since there's no additional information for this event to carry beyond just the `MenuItem`, I don't think we need to include it (or the associated overload). I meant to document this in the README but forgot.